tests/console/yast2_bootloader.pm

```diff
@@ -27,6 +26,7 @@ sub run() {
     script_run("/sbin/yast2 bootloader; echo yast2-bootloader-status-\$? > /dev/$serialdev", 0);
     assert_screen "test-yast2_bootloader-1", 300;
     send_key "alt-o";                                     # OK => Close
+    if (assert_screen 'yast2_bootloader-missing_package') { send_key 'alt-i'; }
```

Comment:
sorry, I prepared an answer by email but seems it got lost. Yes, a check_screen might work but waste time because we don't know if the screen will appear at all or when so I guess it has to be something like this:

```perl
assert_screen([qw(yast2_bootloader-missing_package yast2_bootloader-finished)]);
if (match_has_tag('yast2_bootloader-missing_package')) {
    wait_screen_change { send_key 'alt-i' };
}
```
with "yast2_bootloader-finished looking like https://openqa.opensuse.org/tests/286792#step/yast2_bootloader/4
Does this make sense to you?